tasks/agent.py

```diff
@@ -91,6 +91,9 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
         inv agent.build --build-exclude=systemd
     """
 
+    # bail out if GO111MODULE is set to on
+    go111module_envvar("process-agent.build")
```

Comment:
```suggestion
    go111module_envvar("agent.build")
```

(fwiw, haven't found a way to do introspection on the current task to grab the task name programmatically...)